docs/concepts/json.md

```diff
@@ -60,6 +60,62 @@ with one noticeable enhancement being that `jiter` supports deserialization of `
 In the future, `jiter` is intended to enable support validation errors to include the location
 in the original JSON input which contained the invalid value.
 
+### Partial JSON Parsing
+
+Starting in v2.7.0 and above, Pydantic's [JSON parser](https://docs.rs/jiter/latest/jiter/) offers support for partial JSON parsing, which is exposed via [`pydantic_core.from_json`][pydantic_core.from_json]. Here's an example of this feature in action:
+
+```py
+from pydantic_core import from_json
+
+partial_json_data = '["aa", "bb", "c'  # (1)!
+
+try:
+    result = from_json(partial_json_data, allow_partial=False)
+except ValueError as ve:
+    print(ve)  # (2)!
+    #> EOF while parsing a string at line 1 column 15
+
+result = from_json(partial_json_data, allow_partial=True)
+print(result)  # (3)!
+#> ['aa', 'bb']
+```
+
+1. The JSON list is incomplete - it's missing a closing `"]`
+2. When `allow_partial` is set to `False`, a parsing error occurs.
+3. When `allow_partial` is set to `True`, part of the input is deserialized successfully.
+
+This also works for deserializing partial dictionaries. For example:
+
+```py
+from pydantic_core import from_json
+
+partial_dog_json = '{"breed": "lab", "name": "fluffy", "friends": ["buddy", "spot", "rufus"], "age}'
+dog_dict = from_json(partial_dog_json, allow_partial=True)
+print(dog_dict)
+#> {'breed': 'lab', 'name': 'fluffy', 'friends': ['buddy', 'spot', 'rufus']}
+```
```

Comment:
Probably worth adding an example of 

```py
MyModel.model_validate(pydantic_core.from_json(data, allow_partial=True))
```

Reply:
We also need to explain that:
1. Every field on the model needs to have a default for this to work reliably
2. You need to set `on_error='omit'` in the core schema for types that can fail.

Reply:
Done, good idea, thanks!